plugins/aea-test-autonomy/aea_test_autonomy/docker/tendermint.py

```diff
@@ -134,28 +133,6 @@ def __init__(  # pylint: disable=too-many-arguments
     ):
         """Initialize."""
         super().__init__(client, abci_host, abci_port, port, p2p_port, com_port)
-        self.__create_flask_tendermint_image()
-
-    def __create_flask_tendermint_image(self) -> None:
-        """Create an image of the Flask server with Tendermint."""
-        try:
-            self._client.images.get(self.tag)
-        except ImageNotFound:
-            # TOFIX - remove.
-            cwd = os.getcwd()
-            current_file_folder = os.path.dirname(os.path.realpath(__file__))
-            root = current_file_folder.split(os.path.sep)[:-3]
-            os.chdir(os.path.join(os.path.sep, *root))
-            cmd = [
-                "autonomy",
-                "deploy",
-                "build",
-                "image",
-                "valory/oracle_hardhat",
-                "--dependencies",
-            ]
-            subprocess.run(cmd)  # nosec   # pylint: disable=subprocess-run-check
-            os.chdir(cwd)
```

Comment:
Removed since we now do not run the tests with local images' versions.

Reply:
@angrybayblade safe to remove?

Reply:
It is safe because we now pull from the hub if the tag is not found locally. Either way, the command had not been updated and does not work anymore (`--dependencies` does not exist). It would fail and since there is no check for the return code, we would never know.